R/visualizations.R

```diff
@@ -111,6 +108,16 @@ plotCellVsReference <- function(test, test.id, ref, ref.id, assay.type.test = 'l
 #' # Creating a heatmap with clusters.
 #' plotScoreHeatmap(pred, clusters=test$label)
 #' 
+#' #To only show certain labels, you can use labels.use or max.labels
```

Comment:
Space between hash and first letter, for readability.

Reply:
Added spaces after all 3 # without spaces after them.